runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/ActiveWorkState.java

```diff
@@ -83,6 +85,33 @@ static ActiveWorkState forTesting(
     return new ActiveWorkState(activeWork, computationStateCache);
   }
 
+  private static Stream<KeyedGetDataRequest> toKeyedGetDataRequests(
```

Comment:
nit: this is just moved, but since further from context now how about a better name capturing that this is for heartbeats

makeHeartbeatKeyedGetDataRequests?

Reply:
done

Reply:
are you missing a push? not seeing this change and others below